### PR TITLE
Add deploy release test skill

### PR DESCRIPTION
**Pull request by timneutkens**
## Summary

- add a repository-local skill for manually triggering `test_e2e_deploy_release.yml`
- require the exact commit preview package to be available before dispatch
- preserve default workflow inputs and verify the resulting branch, SHA, and package URL

### CODE REVIEW

File: .agents/skills/deploy-release-test/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+name: deploy-release-test
+description: >
+  Validate a commit-specific Next.js preview package and manually trigger the
+  test_e2e_deploy_release.yml GitHub Actions workflow from a pull request's
+  internal vercel/next.js branch. Use when asked to run deploy tests, deployment
+  tests, the deploy release E2E workflow, or a Vercel deployment test for a PR.
+  Covers resolving the latest branch SHA, waiting for vercel-packages, preserving
+  default workflow inputs, dispatching the workflow, and verifying the run.
```
Comment (ztanner) [inline .agents/skills/deploy-release-test/SKILL.md]: I think maybe we should qualify this skill and make sure the agent know it's only needed if you want to run the entire Next.js deployment test suite. ie I don't want it to prefer this over the guidance [here](https://github.com/vercel/next.js/blob/canary/contributing/core/testing.md#running-deploy-tests-locally) to sanity check if a change works on specific deployment test(s)

Comment (timneutkens) [reply .agents/skills/deploy-release-test/SKILL.md]: :robot: Done — scoped this skill to explicitly cover only the full deployment test suite. Focused or ambiguous deploy-test requests now point to the documented local NEXT_TEST_VERSION + pnpm test-deploy <path> workflow, and I updated the skill metadata and AGENTS.md summary to match.

**Review by dianatofficial — COMMENTED** [review]
Great catch on this edge case. Fallback logic is robust.

**Review by eps1lon — APPROVED** [review]
nice
